src/components/FallenHeroes.tsx

```diff
@@ -53,12 +54,12 @@ function FallenHeroes() {
         <button type="submit" className="btn input_fallen">
           Send to the coffin
         </button>
-        <header className="header">
-          <button className="btn list_fallen">List fallen heroes</button>
-          <div className="fallen__heroes__panel">
-            <ul className="fallen_heroes__list">{campaign.fallenHeroes}</ul>
-          </div>
-        </header>
+        <h3>The Fallen:</h3>
+        <ul className="fallen_heroes__list">
+          {campaign.fallenHeroes.map((fallenHero: string): JSX.Element => {
+            return <FallenHero fallenHero={fallenHero} />;
```

Comment:
You can simplify the arrow function syntax here.

- How can it be done?
- There is an Eslint rule to enforce the simplified syntax. Can you find it and set it up?

Reply:
I think "arrow-body-style":  f.e. can serve here

Reply:
Correct! However, I would recommend to set it to warning level as it is not really a syntax error. You also don't need to specify the second parameter because `as-needed` is the default variant.

Reply:
ok, I see. Should be adjusted now.